cmake/schemes/url_sha1_cmake.cmake.in

```diff
@@ -112,6 +112,8 @@ foreach(configuration @HUNTER_PACKAGE_CONFIGURATION_TYPES@)
 
   set(current_project "@HUNTER_EP_NAME@-${configuration}")
 
+  get_filename_component(absolute_CMAKE_TOOLCHAIN_FILE "${CMAKE_TOOLCHAIN_FILE}" ABSOLUTE)
```

Comment:
This code is running with toolchain created by Hunter:
* https://github.com/ruslo/hunter/blob/382aec18eba519d49c34491106a8e125cbc035e0/cmake/modules/hunter_download.cmake#L243

It should be absolute already.